orbital4c/orbital.py

```diff
@@ -40,6 +40,18 @@ def __sub__(self, other):
         output.comp_array = self.comp_array - other.comp_array
         return output
 
+    def save(self, name):
+        self.comp_array[0].save(f"Large_alpha_{name}")
+        self.comp_array[1].save(f"Large_beta_{name}")
+        self.comp_array[2].save(f"Small_alpha_{name}")
+        self.comp_array[3].save(f"Small_beta_{name}")
+
+    def read(self, name):
+        self.comp_array[0].read(f"Large_alpha_{name}")
+        self.comp_array[1].read(f"Large_beta_{name}")
+        self.comp_array[2].read(f"Small_alpha_{name}")
```

Comment:
I would possibly have done the other way around (`name_large_alpha_real` instead of `real_Large_alpha_name` but it is OK :-)